api/v1/controllers/botData.js

```diff
@@ -13,12 +13,14 @@
 
 const helper = require('../helpers/nouns/botData');
 const Bot = require('../../../db').Bot;
+const BotData = require('../../../db').BotData;
```

Comment:
Can you use the BotData model that has already been required in `nouns/botData`?